index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare type Comparator = (objectA, objectB) => boolean;
+
+export declare function createCustomEqual(createIsEqual?: Comparator): Comparator; 
+export declare function deepEqual(objectA, objectB): boolean;
+export declare function shallowEqual(objectA, objectB): boolean;
+
+export default {
+    createCustom: createCustomEqual,
+    deep: deepEqual,
+    shallow: shallowEqual,
+};
```

Comment:
My question remains (and its genuinely a question, because I don't know TypeScript) ... you define `export default` in `index.d.ts` for the three methods, but there is no actual default export (all three methods are named exports). meaning:

```javascript
// this will not work
import fe from 'fast-equals';

// this is how you get that namespaced object with all three methods
import * as fe from 'fast-equals';
```

I just want to make sure that the typings you provide here are valid for that. I feel like it may not be, because it establishes a contract for a default export that doesn't exist, but I could totally be wrong so I'm just asking you because you would know better than me haha.